src/ucp/core/ucp_worker.c

```diff
@@ -414,8 +414,6 @@ static unsigned ucp_worker_iface_err_handle_progress(void *arg)
         }
     }
 
-    ucp_stream_ep_cleanup(ucp_ep);
```

Comment:
where is it called now?

Reply:
`ucp_ep_disconnected` below or by `ucp_ep_close_nb`